llvm/lib/Target/X86/MCTargetDesc/X86MCCodeEmitter.cpp

```diff
@@ -111,9 +111,9 @@ class X86OpcodePrefixHelper {
   //  0b11: F2
 
   // EVEX (4 bytes)
-  // +-----+ +---------------+ +--------------------+ +------------------------+
-  // | 62h | | RXBR' | B'mmm | | W | vvvv | X' | pp | | z | L'L | b | v' | aaa |
-  // +-----+ +---------------+ +--------------------+ +------------------------+
+  // +-----+ +---------------+ +-------------------+ +------------------------+
+  // | 62h | | RXBR' | B'mmm | | W | vvvv | U | pp | | z | L'L | b | v' | aaa |
```

Comment:
Can we use `X'/U` here as we have interface `setXX2`

Reply:
That will result in out of 80 columns.

Reply:
Besides, I assume the SDM will only use `U` for this bit.

Reply:
Out of 80 columns is not an issue. We can use sth like
```
// clang-format off
// clang-format on
```
for the comment.

Reply:
> Besides, I assume the SDM will only use `U` for this bit.

Fine. I believe you will add X' back if it's added into SDM?

Reply:
Sure.